lib/apipie/error_description.rb

```diff
@@ -17,9 +17,13 @@ def initialize(code_or_options, desc = nil, options = {})
         @metadata = code_or_options[:meta]
         @description = code_or_options[:desc] || code_or_options[:description]
       else
-        @code = 
+        @code =
           if code_or_options.is_a? Symbol
-            Rack::Utils::SYMBOL_TO_STATUS_CODE[code_or_options]
+            begin
+              Rack::Utils.status_code(code_or_options)
+            rescue ArgumentError
+              nil
```

Comment:
why `nil` here? instead of `code_or_options` ?

Reply:
Preserves the old behaviour - if `code_or_options` was a symbol and that symbol wasn't present in `Rack::Utils::SYMBOL_TO_STATUS_CODE` then the return would be `nil`.  Happy to change to `code_or_options` though if you think that's better - arguably it'd probably be more useful to have `code_or_options` and it's closer to the "not a symbol" code path.